docs/database/includes/cosmos-app-host.md

```diff
@@ -129,6 +129,9 @@ var cosmos = builder.AddAzureCosmosDB("cosmos-db")
 
 When you call `AddDatabase`, it configures your Cosmos DB resources to have a database named `db`. The database is created in the Cosmos DB account that's represented by the `AzureCosmosDBResource` that you added earlier. The database is a logical container for collections and users. For more information, see [Databases, containers, and items in Azure Cosmos DB](/azure/cosmos-db/resource-model).
 
+> [!NOTE]
+> When using the `AddDatabase` API to add a database to an Azure Cosmos DB resource, if you're running the emulator, the database isn't actually created in the emulator. This API is intended to include a database in the [Bicep generated](#generated-provisioning-bicep) by the provisioning infrastructure.
```

Comment:
This is true for *all* database resources

Reply:
That's really good to know... it's certainly confusing, I'll make it a common blub to share.

Reply:
- https://github.com/dotnet/aspire/issues/4813
- https://github.com/dotnet/aspire/issues/6863

We're going to end up fixing it so that it does create the database.